Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -355,9 +355,20 @@ namespace AZ::Reflection
                         }
                     }
                 }
-
                 CacheAttributes();
 
+                // Inherit the change notify attribute from our parent
+                const Name changeNotify = Name("ChangeNotify");
+                Dom::Value changeNotifyValue = Find(changeNotify);
+                if (changeNotifyValue.IsNull())
+                {
+                    changeNotifyValue = Find(Name(), changeNotify, parentData);
```

Comment:
This probably just shows my lack of knowledge surrounding groups, but if we always pass an empty name for the group here then do we need the `Name group` param in the new `Find` function?

If this function is adding missing utility and the group is useful elsewhere then ignore this comment!

Reply:
The new Find function can be used to return the Dom::value of any attribute, not just the ChangeNotify, so I opted to include the `Name group` param to keep consistency with the other Find function. 

For the ChangeNotify attribute I noticed that the group name was always an empty string, so that's why I pass an empty name in this instance